web-next/app/agent/multi-agent-orchestration/page.module.css

```
@@ -0,0 +1,531 @@
+  --color-text-tertiary: #6c7480;
+  --color-text-info: #7cc3ea;
+  --color-text-danger: #e5908c;
+  --color-text-success: #85d3a8;
+  --color-text-warning: #e3c576;
+
+  --color-border-primary: #2c313a;
+  --color-border-secondary: #232830;
+  --color-border-tertiary: #1a1e24;
+
+  --font-sans: "Hiragino Kaku Gothic ProN", "Noto Sans JP", "Segoe UI", -apple-system, sans-serif;
+  --font-mono: var(--font-mono, "JetBrains Mono"), "SFMono-Regular", Consolas, monospace;
```

Stylelint (17.14.1)

[error] Expected "Consolas" to be "consolas" (value-keyword-case)

(value-keyword-case)